src/multivector/seq_multivector.c

```diff
@@ -26,7 +26,7 @@ hypre_SeqMultivectorCreate( HYPRE_Int size, HYPRE_Int num_vectors  )
 {
    hypre_Multivector *mvector;
 
-   mvector = (hypre_Multivector *) hypre_MAlloc(sizeof(hypre_Multivector), HYPRE_MEMORY_HOST);
+   mvector = hypre_TAlloc(hypre_Multivector, 1, HYPRE_MEMORY_HOST);
```

Comment:
Why are the TAlloc's needed here?

Reply:
We want to use `hypre_T*` macros, so we can track all the memory operations using the tracker.

Reply:
If we find the tracker useful enough, we should always use `hypre_T*` macros to do the memory operations within hypre. There are a few places I mentioned in this PR that I don't know how to use the macros because of some C syntax, where I manually add the tracker insertions.

Reply:
Nevermind.  I confused myself about what TAlloc does, thinking that it was CTAlloc.  I don't think we should ever use Malloc directly anywhere in the code if we can help it.

Reply:
OK. Thanks! Btw, this the place where I don't know how to use TAlloc. https://github.com/hypre-space/hypre/pull/741/files#r988392174